### PR TITLE
fix: Custom mod overlays do not render on chat

### PR DESCRIPTION
**Pull request by ryanzhoudev**
POST does not render behind chat

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java [hunk 1]
```diff
@@ -19,5 +19,8 @@
 
     RenderEvent.ElementType renderType();
 
-    RenderState renderAt() default RenderState.POST;
+    RenderState renderAt() default RenderState.PRE;
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: I surely don't want this to be changed.. We've set every overlay with the assumption this is Post, and specified PRE for specific cases. I've seen the chat problem, and most likely a better fix would be is to just patch the chat background to not be "solid".

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: Even if it seems working, we would have to test tens of overlays manually, and make sure it does not interact weirdly with anything, which is problematic, and WILL lead to bugs.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: Not to mention, this does not fix any overlays' rendering issues that need to be rendered on Post for some reason.

File: common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2022-2023.
+ * Copyright © Wynntils 2022-2024.
```
Comment (magicus) [inline common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: Revert this change, since the file is not edited.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: Because of the revert commit, this will be added back.

Comment (magicus) [reply common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: Is that so? I thought spotless just diffed the file, but you are saying that it looks throught the commit history?

Comment (magicus) [inline common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]:
```suggestion
 * Copyright © Wynntils 2022-2023.
```

Comment (magicus) [reply common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: Let's see what happens! :)

**Review by magicus — APPROVED** [review]

File: common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java (hunk 2 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java]: Seems like the undoing stuck.
